Source/scrollrt.cpp

```diff
@@ -212,8 +212,8 @@ void DrawDeadPlayer(int x, int y, int sx, int sy, int CelSkip, int CelCap, BOOL
 			}
 			nCel = p->_pAnimFrame;
 			pFrameTable = (DWORD *)pCelBuff;
-			if (nCel < 1 || pFrameTable[0] > 50 || nCel > (int)pFrameTable[0]) {
-				// app_fatal("Drawing dead player %d \"%s\": facing %d, frame %d of %d", i, p->_pName, p->_pdir, nCel, pFrameTable[0]);
+			if (nCel < 1 || SDL_SwapLE32(pFrameTable[0]) > 50 || nCel > (int)SDL_SwapLE32(pFrameTable[0])) {

```

Comment:
For this one, the Mac port appears to save it to a variable and only do the swap once that way.